client/src/main/scala/client/SubmitSolutionClient.scala

```diff
@@ -21,16 +21,20 @@ object SubmitSolutionClient extends JSApp {
   def main(): Unit = initSubmitter("submit", "solution", to = "report")
 
   def initSubmitter(buttonId: String, solutionId: String, to: String) = {
+    val submitButton = jQuery(s"#$buttonId")
+    submitButton.click(submit _)
+
+    def enableButton(): Unit = submitButton.prop("disabled", false); ()
```

Comment:
Format code and you will see that () is out of `enableButton` scope
>     def enableButton(): Unit = submitButton.prop("disabled", false);
>     ()

but I suggest simply change to
> def enableButton(): Unit = submitButton.prop("disabled", false)